### ISSUE TITLE
Confused with the driver installer

### ISSUE THREAD

**Issue by step2ice** [nefarius/ScpToolkit#118]
I don't really undertsand how the driver installer works.

Basically I should install a Xbox360 driver for PS3 / 4 Pads on the PC. Problem is all are shown in the list wheter they are connected at the moment or not.

So I don't know which is the correct one.

![02](https://cloud.githubusercontent.com/assets/16519809/12075874/71522caa-b190-11e5-9f20-f0059343102f.jpg)
![01](https://cloud.githubusercontent.com/assets/16519809/12075875/7153f9b8-b190-11e5-83e3-5faaa3843897.jpg)

So how can I find out which is the right pad to use?

I already installed the Playstation 3 driver and it says it is connected but when I look in the 
input tester, no button press is shown.
![03](https://cloud.githubusercontent.com/assets/16519809/12075882/ba1a1fc4-b190-11e5-957d-11bc79200329.jpg)

Any help is appreciated.

**Comment by mirhl** [comment]
For god's sake [uninstall motionjoy](http://pcgamingwiki.com/wiki/Controller:DualShock_3#Removing_MotionInJoy) before all.

**Comment by step2ice** [comment]
Yeah I did that with help of the tutorial found here http://emulation-general.wikia.com/wiki/MotioninJoy#Un-Installation_instructions

Unfortunately I'm unable to remove the specific components shown in the driver installer.
I can try the geek unisntaller but the rest of the steps are also in the tutorial mentioned above.

**Comment by nefarius** [comment]
Ok what am I missing here? After reading the thread multiple times I don't understand your question? Enlighten me! :)

**Comment by step2ice** [comment]
Ok, so yeah, it's a bit confusing.  :p

I'll try to start over.

1. I deleted Motionjoy with this tutorial http://emulation-general.wikia.com/wiki/MotioninJoy#Un-Installation_instructions

2. I installed the SCP package like it is discribed here https://github.com/nefarius/ScpServer/blob/master/README.md#installation-how-to

-> At step 5 it says "When running the Driver Installer, choose your Bluetooth and controller devices you like to use with ScpToolkit."

And that's exactly the problem question. I got like 20 entries in the list and about 5 or six make sense since they have the word controller in their description.

I tried the entry / check box which reads "Playstation(R) 3 controller".

SCP gives the information (baloon pop ups) that everything went fine.
BUt when I test the controller with the SCP input test or ingame nothing happens.

So the questions would be:
1. What went wrong or what could be the reason for my controller not working since I did everything like it is discribed in the tutorial.

2. In the driver Installer under the point "Install dual shock 3 driver" which entry should be there when you connect a PS3 controller via USB to your PC? -> I can find and tested "Playstation(R) 3 controller" but that didn't work as described above.

**Comment by mirhl** [comment]
You didn't delete motionjoy if it's still shown between connected devices. 
Try to go in device manager and Uninstall "delete driver" the thing.

**Comment by step2ice** [comment]
> You didn't delete motionjoy if it's still shown between connected devices. 
Try to go in device manager and Uninstall "delete driver" the thing.

Well I deleted the oem.inf files, did the cmd commands and removed it from my programms via uninstall.

I believe the stuff which shows up are remaining registry entries from MJ nothing more.

I even looked in the device manager as you said. There are no remains of MJ left, only gamecontroller / drivers by Microsoft.

So I don't think thats the problem here.

**Comment by mirhl** [comment]
If the driver installer is reading *MotionJoy Virtual Xinput Device for Windows* that means device is being reported as such. And if this happens it means it's still using one of the mj .infs which is no good sign.

**Comment by nefarius** [comment]
@mirhl haha indeed it does :D didn't even see it!

**Comment by step2ice** [comment]
>If the driver installer is reading MotionJoy Virtual Xinput Device for Windows that means device is being reported as such. And if this happens it means it's still using one of the mj .infs which is no good sign.

Well, how can I remove the remains? I did it like it's described in the tutorial.
If you have an idea please let me know. :)

**Comment by mirhl** [comment]
Check device manager for wherever the hell that entry is. 
Press uninstall driver, with the checkbox "delete driver" ticked.

**Comment by step2ice** [comment]
>Check device manager for wherever the hell that entry is. 
Press uninstall driver, with the checkbox "delete driver" ticked.

Searched the whole thing, believe it or not, there is no MJ left in there.

I managed to get the DS3 working (for a few days at least).

I ticked Install DS3 driver (for PS3 gamecontroller) and unchecked everything else.

Just recently I wanted to use the controller again and now it stopped working. 

I tried to reinstall everything as I did before but no luck.

The service settings of SCP also don't work most of the time and give a error at startup.
Although systemwise nothing chanced.

I really don't know what the cause of this strange behaviour is.

Maybe there is a more stable version of SCP and windows 7 for me to use?
